filesystem/File.php

```diff
@@ -390,7 +390,6 @@ public function getCMSFields() {
 
 		//get a tree listing with only folder, no files
 		$folderTree = new TreeDropdownField("ParentID", _t('AssetTableField.FOLDER','Folder'), 'Folder');
-		$folderTree->setChildrenMethod('ChildFolders');
```

Comment:
Still doing it, we're just doing it automatically in TreeDropDownField when the source class is set to Folder. Leaving this line would be harmless but it's redundant.